string                      agent_name
string                      role
mac_ros_bridge/Position     pos
string                      assisting_agent
bool                        active